Sources/TuistDependencies/SwiftPackageManager/SwiftPackageManagerInteractor.swift

```diff
@@ -199,6 +204,17 @@ public final class SwiftPackageManagerInteractor: SwiftPackageManagerInteracting
     }
 }
 
+extension TuistGraph.Package {
```

Comment:
This method is not scoped to the interactor so I think it makes sense for it to be in `Package.swift`. That way we can reuse it from other places if needed. I'd add a test to it.